Makefile

```
@@ -5,54 +5,62 @@ VALUES_FILE ?= values/development/values.yaml
-	./scripts/setup.sh
-
-.PHONY: teardown
-teardown: ## Teardown OSAC deployment
-	./scripts/teardown.sh
-
 ##@ Validation
 
 .PHONY: helm-validate
-helm-validate: helm-lint ## Validate Helm chart (lint + template)
-	helm template osac charts/osac/ --values $(VALUES_FILE) > /dev/null
+helm-validate: helm-lint ## Validate all charts (lint + template)
```

checkmake (0.3.2)

[warning] Required target "all" is missing from the Makefile.

(minphony)